exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/ColumnType.kt

```diff
@@ -546,6 +546,12 @@ abstract class StringColumnType(
     /** Returns the specified [value] with special characters escaped. */
     protected fun escape(value: String): String = value.map { charactersToEscape[it] ?: it }.joinToString("")
 
+    /** Returns the specified [value] with special characters escaped and wrapped in quotations, if necessary. */
+    protected fun escapeAndQuote(value: String): String = when (currentDialect) {
+        is PostgreSQLDialect -> "\"${escape(value)}\""
+        else -> escape(value)
+    }
```

Comment:
This logic could not be applied directly to `escape()` because `nonNullValueToString()` is dependent on it.

The quotation is only necessary in `sqlType()` overrides.